src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, {useState, useEffect} from 'react';
+import { Routes, Route } from 'react-router-dom';
+import { Main, Repo } from './pages'
+// import { UserInfo } from './components';
+import axios from 'axios';
+
+
+function App() {
+    const [ username, setUsername ] = useState();
+    const [repos, setRepos] = useState();
+    const [selectedRepo, setSelectedRepo] = useState()
+
+    useEffect(async () => {
+        try {
+            let {data} = await axios.get(`https://api.github.com/users/${username}/repos`)
```

Comment:
Nice - but this only gets at most 30 back, what if you want to find the repos of a user with 100 repos?

Reply:
Good point! Any suggestion?